pydis_site/apps/resources/views/resources_list.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from typing import Any, Dict
+
+import yaml
+from django.conf import settings
+from django.http import Http404
+from django.views.generic import TemplateView
+
+RESOURCES_PATH = Path(settings.BASE_DIR, "pydis_site", "apps", "resources", "resources")
+
+
+class ResourcesList(TemplateView):
+    """Shows specific resources list."""
+
+    template_name = "resources/resources_list.html"
+
+    def get_context_data(self, **kwargs) -> Dict[str, Any]:
+        """Add resources and subcategories data into context."""
+        context = super().get_context_data(**kwargs)
+
+        resource_path = RESOURCES_PATH / self.kwargs["type"]
```

Comment:
Since we refer to it everywhere else as `category`, shouldn't this be `category` rather than `type`?